import lodestone

class CustomException(Exception):
    pass

# why